ecosystem/indexer/src/main.rs

```diff
@@ -76,6 +76,7 @@ async fn main() -> std::io::Result<()> {
         tailer.run_migrations();
     }
 
+    tailer.check_or_update_chain_id().await.unwrap();
```

Comment:
Don't unwrap, use `?`

Reply:
`tailer.check_or_update_chain_id().await?;` gives me this error: 

`required because of the requirements on the impl of `FromResidual<Result<Infallible, anyhow::Error>>` for `Result<(), std::io::Error>``